ui/client/components/AddProcessDialog.js

```diff
@@ -119,12 +120,12 @@ const nameAlreadyExists = (clashedNames, name) => {
 
 const validators = [
   {
-    isValid: (clashedNames, name) => notEmptyValidator.isValid(name),
-    message: notEmptyValidator.message,
+    isValid: (clashedNames, name) => mandatoryValueValidator.isValid(name),
+    message: mandatoryValueValidator.message,
   },
   {
     isValid: (clashedNames, name) => !nameAlreadyExists(clashedNames, name),
-    message: duplicateValue,
+    message: i18next.t("validation.duplicateValue", "This value is already taken"),
```

Comment:
i18next.t Is called once, it'll not react to lang change or lazy loaded dicts.

Reply:
I've moved it to DialogMessages for now